Tasks/AndroidSigning/package.json

```diff
@@ -17,6 +17,6 @@
   },
   "homepage": "https://github.com/microsoft/vsts-tasks#readme",
   "dependencies": {
-    "vsts-task-lib": "^0.7.4"
+    "vsts-task-lib": "^2.1.0"
```

Comment:
My recommendation (after being burned multiple times) is to pin to a specific version.